### ISSUE TITLE
Small fix for admin js

### ISSUE THREAD

**Issue by dklimov1** [revive-adserver/revive-adserver#701]
Actual for disabled Conversion Tracking

```
--- www/admin/assets/js/ox.ui.ORIG.js   2016-04-01 08:47:14.188472874 +0300
+++ www/admin/assets/js/ox.ui.js        2016-04-01 08:48:21.456472805 +0300
@@ -458,7 +458,7 @@
     //update fields states to reflect current values
     formFormat($impressionsField.get(0), true);
     formFormat($clicksField.get(0), true);
-    if ($conversionsField) {
+    if ($conversionsField.length) {
         formFormat($conversionsField.get(0), true);
     }

```

**Comment by justb81** [comment]
Could you make a PR for this?

Description for the fixed behavior:
If one disables Conversion Tracking, the Campaign-Edit-View throws a `TypeError: Cannot read property 'value' of undefined`, because fields are missing.

**Comment by klimovdm** [comment]
I found PR by @FirstImpression 
https://github.com/revive-adserver/revive-adserver/pull/808/files

**Comment by justb81** [comment]
see #808

**Comment by andrewatfornax** [comment]
Fixed via #808.
